torch/nn/modules/module.py

```diff
@@ -702,6 +702,28 @@ def double(self: T) -> T:
         """
         return self._apply(lambda t: t.double() if t.is_floating_point() else t)
 
+    def cfloat(self: T) -> T:
```

Comment:
Mmm I don't know if we wanna add this for nn modules.

Reply:
Do we support autograd for C->C functions?

Reply:
> Do we support autograd for C->R functions?

yeah

Reply:
I don't think we should add that for nn.Module either.
This would lead to partially initialized weights which is not the expected behavior for all the other similar functions.

From previous discussion, we want to get more data on how people use nn.Module for complex before choose what will be the behavior of such function on nn.Modules